.travis.yml

```diff
@@ -1,14 +1,10 @@
 language: node_js
 sudo: false
 node_js:
-- '4'
-# AWS Lambda
-- '4.3.2'
```

Comment:
Removing those would make this semver-major.

Reply:
I added 4 and 4.3.2 on my local branch. But error on 4.3.2.
https://travis-ci.org/redboltz/MQTT.js/builds/534428335